src/components/application_manager/src/mobile_command_factory.cc

```diff
@@ -702,6 +704,17 @@ CommandSharedPtr MobileCommandFactory::CreateCommand(
           message, application_manager);
       break;
     }
+    case mobile_apis::FunctionID::SendHapticDataID: {
+      if ((*message)[strings::params][strings::message_type] ==
+          static_cast<int>(application_manager::MessageType::kResponse)) {
+        command.reset(
+            new commands::SendHapticDataResponse(message, application_manager));
+      } else {
+        command.reset(
+            new commands::SendHapticDataRequest(message, application_manager));
+      }
+      break;
+    }
```

Comment:
To maintain consistency this should be moved to the `request/response` section, right below `DialNumber`.